### PR TITLE
(PDB-1890) Add release notes for 3.1.0

### PR DESCRIPTION
**Pull request by senior**

### CODE REVIEW

File: documentation/release_notes.markdown [hunk 1]
```diff
@@ -10,6 +10,64 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+ * Upgrade to clojure.java.jdbc 0.3.7
+   ([PDB-1739](https://tickets.puppetlabs.com/browse/PDB-1739))
+
+ * Switched to the Puppet Labs version-checking library
+   ([PDB-1823](https://tickets.puppetlabs.com/browse/PDB-1823))
+
+ * Cache database metadata on startup. Previously code involving query
+   features that were different between HyperSQL and PostgreSQL would
+   require a database connection to determine the correct SQL to be
+   generated. This changes that to acquire this database metadata on
```
Comment (ajroetker) [inline documentation/release_notes.markdown]: => `that code`

File: documentation/release_notes.markdown [hunk 2]
```diff
@@ -10,6 +10,64 @@ canonical: "/puppetdb/latest/release_notes.html"
...
+   PuppetDB. With this change the data is sent in a format similar to
+   Puppet's internal representation of the data.
+
+ * Added an experimental command parameter
+   "secondsToWaitForCompletion" that allows command POSTs to block up
+   to a specified number of seconds waiting for the command to be
+   successfuly stored.
+
+ * Added realistically spaced timestamps to the benchmark
+   tool. Previously the same time was used for all submitted commands.
```
Comment (ajroetker) [inline documentation/release_notes.markdown]: Also new uuids

**Comment by senior** [comment]
@ajroetker fixed up the release notes based on your suggestions

**Comment by ajroetker** [comment]
Sweet, I'm :+1:, can I go ahead and click merge?

**Pull request merged**